lib/src/includes/signed_video_sign.h

```diff
@@ -499,6 +499,22 @@ signed_video_set_recurrence_interval_frames(signed_video_t *self, unsigned recur
 SignedVideoReturnCode
 signed_video_set_sei_epb(signed_video_t *self, bool sei_epb);
 
+/**
+ * @brief Configures Signed Video use golden SEI principle
+ *
+ * The principle of the golden sei sends all information only needed once such as the Public key at
+ * the start of the stream with golden sei after that rest of the SEIs in the stream only include
+ * mandatory information.
```

Comment:
You should mention that it is the user's responsibility to add the golden SEI at the beginning of the stream. To be more precise, it does not have to start with a golden SEI, the only thing that matters is that the first SEI should be a golden SEI.

Also, I don't think it should be feasible to set this on an ongoing session. That has to be clarified in the description and fixed in the code + tests.